paths/git-trouble/fc-scenarios.md

```diff
@@ -32,6 +32,10 @@ main-content: |
 
   That innovative fix was **supposed** to fix everything! Instead your project is a garbage fire and you just want everything to go back to how it was. Have no fear, we can make that happen.  
 
+  ## We :heart: Feedback
+
+    Please take _literally less than a minute_ to let us know what you think of this course. You also have the option to opt-in for updates on future classes. [Start Survey](http://www.surveygizmo.com/s3/3288550/git-reset)!
```

Comment:
@hollenberry "you also have the option to opt-in" sounds redundant. Could we say: "You may also choose to opt-in"? or something along those lines?